tests/test_project.py

```diff
@@ -355,6 +355,110 @@ def test_context(job):
                     project.run()
                 assert os.getcwd() == starting_dir
 
+    def test_cmd_operation_argument_as_command(self):
+        class A(FlowProject):
+            pass
+
+        @A.operation(with_job=True, cmd=True)
+        def test_cmd(joba, jobb="test"):
+            return "echo '{joba} {jobb}' > output.txt"
+
+        project = self.mock_project(A)
+        with _add_cwd_to_environment_pythonpath():
+            with _switch_to_directory(project.root_directory()):
+                with redirect_stderr(StringIO()):
+                    project.run()
+                for job in project:
+                    assert os.path.isfile(job.fn("output.txt"))
+                    with open(job.fn("output.txt")) as f:
+                        lines = f.readlines()
+                        assert f"{job.id} test\n" == lines[0]
+
+    def test_cmd_operation_argument_as_command_invalid(self):
+        class A(FlowProject):
+            pass
+
+        @A.operation(with_job=True, cmd=True)
+        def test_cmd(job):
+            return "echo '{jobabc}' > output.txt"
+
+        project = self.mock_project(A)
+        with _add_cwd_to_environment_pythonpath():
```

Comment:
Maybe we should make one context manager that does all this… at least up to the pytest.raises.